### PR TITLE
Update code

### PR DESCRIPTION
**Pull request by LucasLucioTO**
# Proposed Changes
 - change respository names;
 - add Mapper;
 - create db_migrations file;

### CODE REVIEW

File: src/main/java/styluscar/client/Client.java [hunk 1]
```diff
@@ -1,19 +1,11 @@
 package styluscar.client;
 
-import javax.persistence.Entity;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-import styluscar.client.dto.CreateClientDto;
-import styluscar.client.dto.UpdateClientDto;
+import lombok.*;
```
Comment (kaio-fabiano) [inline src/main/java/styluscar/client/Client.java:3]: Do not use "*" imports

File: src/main/java/styluscar/client/Client.java [hunk 2]
```diff
@@ -1,19 +1,11 @@
...
-import lombok.Setter;
-import styluscar.client.dto.CreateClientDto;
-import styluscar.client.dto.UpdateClientDto;
+import lombok.*;
 
-import javax.persistence.Column;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
```
Comment (kaio-fabiano) [inline src/main/java/styluscar/client/Client.java:5]: Same issue

File: src/main/java/styluscar/client/ClientResource.java [hunk 3]
```diff
@@ -5,32 +5,34 @@
 
 import javax.inject.Inject;
 import javax.transaction.Transactional;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
+import javax.ws.rs.*;
```
Comment (kaio-fabiano) [inline src/main/java/styluscar/client/ClientResource.java:8]: Same issue

File: src/main/java/styluscar/client/ClientResource.java [hunk 4]
```diff
@@ -5,32 +5,34 @@
...
     ClientService service;
 
     @GET
     @Path("/clients")
+    @Produces(MediaType.APPLICATION_JSON)
     public List<Client> getAllClients() {
         try {
-            return this.service.findAllClient();
+           List<Client> clients = this.service.findAllClient();
+           return clients;
```
Comment (Micledson) [inline src/main/java/styluscar/client/ClientResource.java]: You can use `return this.service.findAllCliente();`

File: src/main/java/styluscar/functionary/FunctionaryResource.java [hunk 5]
```diff
@@ -6,44 +6,46 @@
...
+    FunctionaryService service;
 
     @GET
     @Path("/functionarys")
+    @Produces(MediaType.APPLICATION_JSON)
     public List<Functionary> getAllFunctionarys() {
         try{
-            return this.functionaryService.findAllFunctionary();
+            List<Functionary> functionarys = this.service.findAllFunctionary();
+            return functionarys;
```
Comment (Micledson) [inline src/main/java/styluscar/functionary/FunctionaryResource.java]: Use `return this.service.findAllFuncionary();`

File: src/main/java/styluscar/material/MaterialResource.java [hunk 6]
```diff
@@ -6,45 +6,44 @@
...
+    MaterialService service;
 
     @GET
     @Path("/materials")
+    @Produces(MediaType.APPLICATION_JSON)
     public List<Material> listAllMaterials(){
         try {
-            return this.ms.findAllMaterial();
+            List<Material> materials = this.service.findAllMaterial();
+            return materials;
```
Comment (Micledson) [inline src/main/java/styluscar/material/MaterialResource.java]: Use `return this.service.findAllMaterial();`

**Review by Micledson — CHANGES_REQUESTED** [review]
Just some any changes

File: src/main/java/styluscar/client/Client.java (hunk 1 above)
Comment (LucasLucioTO) [reply src/main/java/styluscar/client/Client.java:3]: Done

File: src/main/java/styluscar/client/ClientResource.java (hunk 4 above)
Comment (LucasLucioTO) [reply src/main/java/styluscar/client/ClientResource.java]: done

File: src/main/java/styluscar/functionary/FunctionaryResource.java (hunk 5 above)
Comment (LucasLucioTO) [reply src/main/java/styluscar/functionary/FunctionaryResource.java]: done

File: src/main/java/styluscar/material/MaterialResource.java (hunk 6 above)
Comment (LucasLucioTO) [reply src/main/java/styluscar/material/MaterialResource.java]: done

**Pull request merged**